app/models/micropost.rb

```diff
@@ -1,7 +1,7 @@
 class Micropost < ActiveRecord::Base
   attr_accessible :content, :user_id
 
-  belongs.to :user
+  #belongs_to :user
 
-  validates :content, :length => {:maxium => 140}
+  #validates :content, :length => {:maxium => 140}
```

Comment:
Unsure what is happing here, as we continue to add rspec test, this will be corrected.